package.json

```diff
@@ -29,6 +29,7 @@
     "copy-notices": "node scripts/generators/copy-notices.js",
     "create-component": "yarn nx g @fluentui/workspace-plugin:react-component",
     "create-package": "yarn nx g @fluentui/workspace-plugin:react-library",
+    "create-recipe": "yarn nx g @fluentui/workspace-plugin:recipe-generator",
```

Comment:
hmm not sure if this is good idea, long term we wanna drastically reduce these global scripts as we are moving to nx -> we have all of these generators already there without need to alias

Reply:
sounds good, I'll remove